src/network/networkd-link.c

```diff
@@ -261,6 +261,20 @@ static int link_enable_ipv6(Link *link) {
         return 0;
 }
 
+static bool link_is_bridge_port(Link *link) {
+
+        assert(link);
+
+        if (link->flags & IFF_LOOPBACK)
+                return false;
+
+        if (link->network)
+                if (link->network->bridge)
+                        return true;
```

Comment:
Please see @gdamjan 's test results.
Also see @poettering  comment about which is not in the configuration.

Let's say networkd have not created a bridge `bridge0` and `eth1/eth2` are bridge port.

-  This can happen before networkd starts.
-  After networkd starts you can also create bridge and add ports to it with ip command.